src/index.js

```diff
@@ -92,11 +90,36 @@ module.exports = function validate(config, options = {}) {
     process.exit(1)
   }
 
+  return config
+}
+
+// Easier consumability for require (default use case for non-transpiled webpack configs)
+function validateRoot(config, options = {}) {
+  const {
+    quiet,
+  } = options
+
+  let validationResult,
+    multiValidationResults
+
+  if (Array.isArray(config)) {
+    multiValidationResults = []
+    config.forEach((cfg) => {
+      multiValidationResults.push(
+        validate(cfg, options)
+      )
+    })
+  } else {
+    validationResult = validate(config, options)
+  }
+
   if (!quiet) {
     console.info(chalk.green('[webpack-validator] Config is valid.'))
   }
 
-  return config
+  return validationResult || multiValidationResults
 }
 
-module.exports.Joi = Joi
```

Comment:
Basically the old implementation did two things:

1. `module.exports = function validate(config, options = {}) {`
2. `module.exports.Joi = Joi`

So it attached `Joi` to the function it exports by default. `exports` returns an object` by definition.

Changing the exports the way they were would fix it. The import at test should change too at least.